### PR TITLE
cpu/kinetis_common/ldscripts: rename the memory markers

### PR DESCRIPTION
**Pull request by jfischer-no**
I have renamed the memory markers. Your startup.c should still work. fcfield can be declared in startup.
I consider to define it week and move startup.c to kinetis common, your opinion?
Still WIP, I would like to make it ready tomorrow.

 Why did you do ALIGN (8)?

### CODE REVIEW

**Comment by jfischer-no** [comment]
@gebart Since I was on remodel for ldscript test, I changed startup. #2412

**Comment by jfischer-no** [comment]
@gebart I do not know currently how to make it better with sram_u and sram_l. I set sram_l to zero.
Otherwise, it was all for now to ldscript.

**Comment by jnohlgard** [comment]
Looks good, I don't remember right now why I did align(8), what is the alignment requirements for the DMA module?

File: cpu/kinetis_common/ldscripts/kinetis-base.ld [hunk 1]
```diff
@@ -203,25 +205,25 @@ SECTIONS
...
         . = . + HEAP_SIZE;
-        . = ALIGN(8);
         _heap_end = .;
+        PROVIDE(__heap_max = .);
     } > sram_u
 
     /* stack section */
     .stack (NOLOAD):
     {
-        . = ALIGN(8);
```
Comment (jnohlgard) [inline cpu/kinetis_common/ldscripts/kinetis-base.ld:221]: The stack must be 8-byte aligned:

http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.faqs/ka4127.html

**Comment by jfischer-no** [comment]
@gebart I have changed the stack alignment back to 8. About DMA, I looked over a few Freescale's linker scripts, they align only the stack and heap to 8.

**Comment by jnohlgard** [comment]
Nice! Thanks for the help with RIOT-OS/RIOT#2393

**Pull request merged**